docs/integrations/cloud/azure-app-configuration.mdx

```diff
@@ -0,0 +1,75 @@
+
+        Press on the Azure App Configuration tile and grant Infisical access to App Configuration.
+      </Step>
+      <Step title="Start integration">
+        Obtain the Azure App Configuration endpoint from the overview tab.
+        ![integrations](../../images/integrations/azure-app-configuration/azure-app-config-endpoint.png)
+
+        Select which Infisical environment secrets you want to sync to your Azure App Configuration. Then, input your App Configuration instance endpoint. Optionally, you can define a prefix for your secrets which will be appended to the keys upon syncing.
+
+        ![integrations](../../images/integrations/azure-app-configuration/create-integration-form.png)
+
+        Press create integration to start syncing secrets to Azure App Configuration.
+      </Step>
+    </Steps>
+
+  </Tab>
+  <Tab title="Self-Hosted Setup">
+    Using the Azure App Configuration integration on a self-hosted instance of Infisical requires configuring an application in Azure
+    and registering your instance with it.
+
+    <Steps>
+      <Step title="Create an application in Azure">
+        Navigate to Azure Active Directory > App registrations to create a new application.
+
+        <Info>
+          Azure Active Directory is now Microsoft Entra ID.
+        </Info>
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/config-aad.png)
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/config-new-app.png)
+
+        Create the application. As part of the form, set the **Redirect URI** to `https://your-domain.com/integrations/azure-app-configuration/oauth2/callback`.
+
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/app-registration-redirect.png)
+
+        After registration, set the API permissions of the app to include the following Azure App Configuration permissions: KeyValue.Delete, KeyValue.Read, and KeyValue.Write.
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/app-api-permissions.png)
+
+      </Step>
+      <Step title="Add your application credentials to Infisical">
+        Obtain the **Application (Client) ID** in Overview and generate a **Client Secret** in Certificate & secrets for your Azure application.
+
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/config-credentials-1.png)
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/config-credentials-2.png)
+        ![integrations Azure app config](../../images/integrations/azure-app-configuration/config-credentials-3.png)
+
+        Back in your Infisical instance, add two new environment variables for the credentials of your Azure application.
+
+        - `CLIENT_ID_AZURE`: The **Application (Client) ID** of your Azure application.
```

Comment:
In the self-hosted docs, make sure to mention that the SITE_URL must match the URL being used in the Azure app registration